modules/renter/proto/formcontract.go

```diff
@@ -108,6 +109,7 @@ func (cs *ContractSet) FormContract(params ContractParams, txnBuilder transactio
 	defer func() {
 		if err != nil {
 			hdb.IncrementFailedInteractions(host.PublicKey)
+			err = errors.Extend(err, modules.ErrHostFault)
```

Comment:
hmm. not sure that I like this, because it assumes that any error which occurs later is automatically the host's fault.

It appears to be true here, but it might not be something that we think about correctly if we come back and modify this file later.

Though, it is a protocol file, which means it's not likely to ever be modified. We try not to touch those.

Reply:
no I take it back. Errors can be returned by `AcceptContractSet` and `managedInsertContract` and I'm not sure that either of those errors would indicate the host's fault.

Reply:
In that case we also need to adjust all calls to `IncrementFailedInteractions` since they already assume that it's the hosts fault.

Reply:
Agreed